.github/workflows/docs.yml

```diff
@@ -13,8 +13,14 @@ jobs:
       - name: Checkout the code
         uses: actions/checkout@v2
 
-      - name: Build the docs
-        run: cargo doc --lib --all-features
+      - name: Build the docs (nightly)
+        run: |
+          rustup install nightly
```

Comment:
why aren't we using the same kind of toolchain setup as on CI:
```
 - name: Install latest nightly
      uses: actions-rs/toolchain@v1
      with:
        toolchain: nightly
        override: true
      continue-on-error: true
```

Reply:
Cause I didn't notice that's how the ci did it, will fix.